scrapi/base/schemas.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+from copy import deepcopy
+
+from nameparser import HumanName
+
+
+def default_name_parser(names):
+    names = names.split(';')
```

Comment:
Yeah, I actually wanted your help with this part. I'd like to get all of the harvesters to use this system, but you have a much better understanding of what metadata each service provides. Right now the configuration in this file works for doepages only